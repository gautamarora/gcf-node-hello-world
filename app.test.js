const uuid = require('uuid');
const helloWorld = require('./app').helloWorld

describe('GET /', () => {
  let req
  let res
  beforeEach(() => {
    const name = uuid.v4();
    req = {
      body: {
        name: name
      }
    };
    res = { send: jest.fn() }
  })
  test('should match snapshot', async () => {
      helloWorld(req, res)
      expect(res).toMatchSnapshot()
  });
})
