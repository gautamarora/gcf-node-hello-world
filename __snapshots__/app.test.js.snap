// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET / should match snapshot 1`] = `
Object {
  "send": [MockFunction] {
    "calls": Array [
      Array [
        "Hello World!",
      ],
    ],
    "results": Array [
      Object {
        "isThrow": false,
        "value": undefined,
      },
    ],
  },
}
`;
